src/process/automatic/config.rs

```diff
@@ -29,6 +31,21 @@ pub(crate) struct Config {
     pub monitored_model_failure_msg: String,
 
     pub monitored_msgs_w_trace_lvl: Vec<String>,
+
+    pub gpu_uuid: Option<String>,
+
+    // Property for testing purposes
+    pub uses_gpu: bool,
```

Comment:
I could not solve issue of running functional tests with disabled `uses_gpu`.
The alternative would be to use `gpu-detection` with some Cargo `feature` as a `dev-dependency` which enabled mocked `init` and `detect` functions.